values.yaml

```diff
@@ -0,0 +1,38 @@
+redis:
+  enabled: true
+  auth:
+    enabled: false
+  architecture: standalone
+  master:
+    persistence:
+      enabled: true
+      size: 1Gi  # Adjust the size as needed
+
+onetimesecret:
+  image: 
+    registry: docker.io
+    repository: onetimesecret/onetimesecret
+    tag: latest #will be overwritten by Chart.yaml appVersion
+  service:
+    type: LoadBalancer
+  env:
+    COLONEL: "admin@example.com"
+    SSL: "false"  # Ensure this is a string
+
+### REDIS_URL and HOST Paramerters are dynamically getting updated from templates/onetimesecret-deployment.yaml
+### Expamples for setting SMTP PARAMETERS
+#   SMTP_PORT: "587"
+#   SMTP_USERNAME: "username"
+#   SMTP_PASSWORD: "password"
+#   FROM_EMAIL: "admin@example.com"
+#   TO_EMAIL: "recipent@example.com"
```

Comment:
Per our discussion changing these examples to the same username@example.com ex: no-reply@example.com

Reply:
Like we discussed I removed comments for SMTP and added a sample default values. Just instead of example.com I have added yourdomain.com.